### DISCUSSION TITLE
how to improve the gpu training?

### DISCUSSION THREAD

**Discussion by McAlexDudu** [deepmodeling/deepmd-kit#1565] [General]
Hi, I tried GPU version of deepmd, since I just had one GPU device and only 2GMB memory. But when training, the memory seems use not enough, and I wonder if I could improve the speed?
![截屏2022-03-11 21 18 35](https://user-images.githubusercontent.com/16278257/157873906-7ee57dea-aee3-4cb1-9507-f67420722a20.png)
<img width="882" alt="截屏2022-03-11 21 15 25" src="https://user-images.githubusercontent.com/16278257/157873925-f9f04dcb-e248-4aec-93f9-cccc4c1bedfe.png">

**Comment by njzjz** [comment]
What do you mean when you said the memory not enough? As shown in your snapshot, only 488MB of memory was used.

**Reply by McAlexDudu** [reply]
Sorry, I didn't say it cleary. I mean how to use more GPU memory to impove the speed?

**Reply by njzjz** [reply]
The memory is related to your data size, model size, batch size, etc., but not related to performance.

**Reply by McAlexDudu** [reply]
Got it！
